lib/matplotlib/projections/geo.py

```diff
@@ -497,8 +494,7 @@ def transform_non_affine(self, xy):
             y = xy[:, 1:2]
             clong = self._center_longitude
             clat = self._center_latitude
-            p = np.sqrt(x*x + y*y)
-            p = np.where(p == 0.0, 1e-9, p)
+            p = np.max(np.hypot(x, y), 1e-9)
```

Comment:
shouldn't this be `np.maximum()`?

Reply:
It's a bit scary that we don't error on this, but heh, fixed.